po/eos-desktop.pot

```diff
@@ -1,1350 +0,0 @@
-# SOME DESCRIPTIVE TITLE.
```

Comment:
It looks like this file is being removed by your changes accidentally. Please amend the last commit so the file is not deleted.